views/part-upsell.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use Imagify\User\User;
+
+defined( 'ABSPATH' ) || die( 'Cheatin’ uh?' );
+
+$imagify_user = new User();
+$unconsumed_quota = $imagify_user ? $imagify_user->get_percent_unconsumed_quota() : 0;
+$infinite     = ( 15 === $imagify_user->plan_id || 17 === $imagify_user->plan_id );
```

Comment:
it's not clear what are 15 and 17 representing, Can we plz create new methods inside the User class like is_free one for those plan types and directly use them here? what do u think?